### ISSUE TITLE
Cart feedback notes

### ISSUE THREAD

**Issue by freechocolate** [cbaron/Hazy#16]
Adding feedback notes here.

Product Listings
- Include the name of the manufacturer (Innova, West Side Discs, Disc Mania, etc.)
- Need ability to add "# left in stock" to a DiscType listing (maybe placed next to price)
- Add a "Compare" checkbox for guests to save and later compare attributes of their selection
- Buttons like "See Details" or "Add to Cart" need jazz fingers. Could we start with some shading on the button and changing button background to something other than white?

Search Filters
- Basic searches for discs should be able to filter on DiscType attributes, Weight, and Color
- Advanced filters should include the Disc Attributes like
-- StampType
-- PLH
-- Run
-- TBD

**Comment by freechocolate** [comment]
More Fun

DiscType
- After selecting 'See Discs' on a DiscType, the next view should allow guests to shop the individual discs. Similar to shopping for a laptops and adjust RAM/HDD specs, guests should be able to adjust weight*/color
- Continue to make filters available at this screen. If a guest selects a filter that adds a different DiscType to the viewed selection, can we combine all Discs available from the new DiscTypes into the active view?   
- Ability to "Add to Cart" for Discs should be on this page
- Adding an item to the cart should popup a cart summary notification, but the guest should remain on the page.



Cart Management
- Need easy way to clear all items or group of selected items from the cart

**Comment by freechocolate** [comment]
Following up here on the product listing detail request. Let me know if there are any questions.

DiscType Listing
 - Vendor Name 
 - PlasticName
 - DiscName
 - Flightstat listing
 - DiscClass
 - Price
 - '## Left in stock' (some disctypes will not need this, need ability to not display)
 - 'Compare' checkbox

Disc
 - All DiscType listing attributes minus  '## Left in stock'
 - weight (g)
 - color
 - PLH
> Additional Specs (expandable to view the attributes below)
 - DiscDescription
 - DiscSpeed
 - DiscGlide
 - DiscTurn
 - DiscFade
 - RimDepth
 - RimVAngle
 - SiteUploadDate
 - MFG Birth Date
 - MFG Location
 - Quality
 - DiscHistory
 - PlayerOpinion

**Comment by ScottAP108** [comment]
Hey Mike, Scott here. If you could, please be more specific with regards to layout of attributes for each type of listing. We currently have three types:

1) the disc type listings
2) the horizontal "disc preview" listings that you see when you click on a type listing's "See Discs" button
3) the full disc details listing that will display all of the images/info for a specific disc

For each of those, try to think about not just which attributes you want to display, but also the order in which to display them. I don't have data for all these attributes yet, but it will be helpful to nail this stuff down so I can relay it to Adam so he can create new designs for the listings. Thanks.

**Comment by freechocolate** [comment]
Hey Scott, updates are below. If anything doesn't make sense or needs to be more comprehensive please let me know. I'm a bit uncertain on my #2 response and had some follow-up questions about the view that may change the attributes/order listed. Sorry ahead of time if they're off-topic. Thank you.

1. Disc type listing attribute order
    * disc image
    * mini vendor image + vendor name
    * plastic name + disc name
    * price + (# left in stock)
    * disc class
    * Flightstat listing (Speed | Glide | Turn | Fade)
    * Compare checkbox

2. Horizontal "disc preview" listing attribute order*
    * disc image
    * vendor logo
    * vendor name + plastic name + disc name
    * Weight + Color
    * PLH
    * Price
    * Add to Cart button + See Details button
    * Compare checkbox

3. Full disc details
   * large image
   * thumbnail images
   * mini vendor logo
   * vendor name + plastic name + disc name
   * Weight
   * Color
   * PLH
   * Price
   * (EDIT) manufacturer flightchart 
   * DiscDescription (from the manufacturer)
   * RimDepth + RimVAngle
   * Flightstat listing (Speed | Glide | Turn | Fade)
   * Additional Specs (Click to expand this list)
     * MFG Birth Date
     * MFG Location
     * SiteUploadDate
     * Quality
     * DiscHistory
     * PlayerOpinion


*What is the general idea for this view- will filters still be available? If guests continue to scroll past the list of discs within the current disc type, will the page populate with disc previews of similar disc types?  Do we try to present a disc type's flight chart here (or other static DiscType attributes), but not within the list of discs to reduce displaying the same info for each disc listed?

**Comment by ScottAP108** [comment]
Maybe "preview" is the wrong word. More like a quick summary. The idea is that you will have brief disc listings to give a little bit of info about the disc so the customer can decide on one that looks good and then click to see the full details. if we had full details for every disc from the beginning, it would be a mess. Filters will still be available here, but they will apply to Discs, not Disc Types. This will be useful if you have a whole bunch of discs for the type.

I don't think if you keep scrolling that you should see summary listings for other disc types. That would just be confusing, since you have explicitly selected a disc type on the previous page. If you want to see discs for another disc type, you can click on a "Back to Disc Types" button then select another type.

As you mention, it might not be a bad idea for this page to have all of the disc type's characteristics at the top above the disc summary listings. There will be more space than is available for the disc type listings, so you could be more thorough and add a flight chart or whatever you feel would be good to have.

Let me know if that clarifies things.

One question: you mention vendor for the listings. Why is this vendor and not manufacturer? Your spreadsheet mentioned vendor is who you paid for the disc. But if you have a disc that says "Innova" on it, Innova is the manufacturer, right? Wouldn't that be better to list? Sorry, I'm not a frolfer and all of the semantics are pretty confusing for me.

**Comment by freechocolate** [comment]
Yep that helps. I'll add notes on the #2 DiscType overview layout in case we want to try it out.

> One question: you mention vendor for the listings. Why is this vendor and not manufacturer?

Not all vendors manufacture their own discs. I think Innova made discs for at least 2 other companies who add their stamp and name to the blanks produced from Innova. Haha, yea the semantics still get me, disc manufacturing and selection has changed a lot from where it was 5-10 years. 

Should we consider a 4th view where a guest sees Disc level attributes (like weight, color, PLH) across multiple DiscTypes? I'm thinking of the searcher who wants to view all Discs of the Innova Roc (a mid-range disc), then in the same view be able to adjust the active filter with a different vendors version of that mid-range Roc.
